src/js/components/reviewData/CertificationModal/RevalidateButtons.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+  * RevalidateButtons.jsx
+  * Created by Kevin Li 9/7/16
+  **/
+
+import React from 'react';
+
+export default class RevalidateButtons extends React.Component {
+	render() {
+		return (
+			<div>
+				<div className="row">
+	            	<div className="col-md-12" id="certify-check">
+	            		This submission was created prior to 02/22/2017. Please revalidate this submission before certifying it.
```

Comment:
maybe pass the certify date as a prop for the future so that when we dynamically get the validation date we don't need to keep changing the hardcoded value